support/src/benchmark.rs

```diff
@@ -0,0 +1,35 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2023 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use crate::traits::GetWorstCase;
+
+/// Manly used for the DIP benchmarking. The worst case can be created for the did and submitter.
+pub struct IdentityContext<DidIdentifier, AccountId> {
+	pub did: DidIdentifier,
+	pub submitter: AccountId,
+}
+
+impl<T> GetWorstCase<IdentityContext<T, T>> for u32 {
```

Comment:
`u32` does not depend on DIDs nor accounts, so I would implement the trait in the most generic way possible:

```suggestion
impl<T> GetWorstCase<T> for u32 {
```

Reply:
Also, if we move the `IdentityContext` type where it belongs, this impls can be defined directly where the trait is defined, and there is no need for this `benchmarks` module anymore.

Reply:
I changed the implementations here: [34c2aad](https://github.com/KILTprotocol/kilt-node/pull/585/commits/34c2aadddad67d5cccaf2654ffb2ca1bb6853c64)